LDM-556.tex

```diff
@@ -103,7 +103,7 @@ \subsubsection{Repository Removal}
 \textbf{ID:} DMS-MWBT-REQ-0005 (Priority: 1a)
 
 \textbf{Specification:}
-It shall be possible for an authorized user to remove a DataRepository from any storage environment.
+It shall be possible for an authorized user to remove a Collection from any storage environment.
 
 \textbf{Discussion:}
 Some DataRepositories (e.g. Data Releases) may not have any thusly authorized users. This would also involve removing it from registries. Removing the repository might require database table modifications. (UseCases: LDF3, DRP29)
```

Comment:
Is this `DataRepository` an oversight in the changes, or does it have a different significance?